presto-main/src/test/java/io/prestosql/sql/planner/assertions/Util.java

```diff
@@ -31,6 +31,27 @@
 {
     private Util() {}
 
+    static boolean domainsMatch(TupleDomain<ColumnHandle> expected, TupleDomain<ColumnHandle> actual)
```

Comment:
How does it differ from `io.prestosql.spi.predicate.TupleDomain#equals` (this matches the method name `domainsMatch` or `io.prestosql.spi.predicate.TupleDomain#contains` (this matches the actual code of this method)? What am I missing? Maybe `TupleDomain` should get a new method?